actors/market/tests/market_actor_test.rs

```diff
@@ -729,11 +729,12 @@ fn provider_and_client_addresses_are_resolved_before_persisting_state_and_sent_t
     let end_epoch = start_epoch + 200 * EPOCHS_IN_DAY;
     rt.set_epoch(start_epoch);
 
-    let mut deal = generate_deal_proposal(client_bls, provider_bls, start_epoch, end_epoch);
+    let mut deal =
+        generate_deal_proposal(client_resolved, provider_resolved, start_epoch, end_epoch);
     deal.verified_deal = true;
 
     // add funds for cient using it's BLS address -> will be resolved and persisted
-    add_participant_funds(&mut rt, client_bls, deal.client_balance_requirement());
```

Comment:
Blocking: What's going on here?  I don't think this change should break this test?